_articles/media-production-on-pop.md

```diff
@@ -76,3 +76,9 @@ Install Directly From the Pop!\_Shop
 GIMP is a free and open-source raster graphics editor used for image retouching and editing, free-form drawing, converting between different image formats, and more specialized tasks.
 
 Install Directly From the Pop!\_Shop
+
+### Flowblade
+
+Flowblade is a multitrack non-linear video editor with a set of industry standard editing tools suitable from beginners to masters to help make your vision a reality of image and sound. 
```

Comment:
Thank you for the contribution! The `tools suitable from beginners to masters` part reads a little strangely to me. Perhaps `tools suitable for beginners and masters alike` would sound more natural?

Reply:
Sure thing!